development/docker-compose.base.yml

```diff
@@ -37,3 +38,13 @@ services:
       retries: 3
       test: ["CMD", "bash", "-c", "nautobot-server celery inspect ping --destination celery@$$HOSTNAME"]  ## $$ because of docker-compose
     <<: *nautobot-base
+  beat:
+    entrypoint:
+      - "sh"
+      - "-c"  # this is to evaluate the $NAUTOBOT_LOG_LEVEL from the env
+      - "nautobot-server celery beat -l $$NAUTOBOT_LOG_LEVEL"  ## $$ because of docker-compose
+    depends_on:
+      - "nautobot"
+    healthcheck:
+      disable: true
+    <<: *nautobot-base
```

Comment:
What are we doing with this in the migration? I don't think there are any scheduled jobs.

Reply:
Same as above, I used SSoT as a model for these files. It should probably be updated to `"nautobot-server celery worker -l $$NAUTOBOT_LOG_LEVEL --events" ` to be in line with other plugins.